.circleci/config.yml

```diff
@@ -22,9 +22,31 @@ jobs:
       - run:
           # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
           command: ./ci_support/run_docker_build.sh
+  build__CONDA_R_3.4.1:
```

Comment:
Why wasn't 3.4.1 added to travis.yml?

Reply:
@isuruf I don't know. I ran `conda smithy rerender --commit auto --feedstock_directory r-anomalydetection-feedstock`

Reply:
As a sanity check, I cloned the feedstock again and re-renedered. Once again only circleci was updated. In case it could be useful, below are the software versions I used:

```
$ conda --version
conda 4.3.29
$ conda build --version
conda-build 2.1.17
$ conda smithy --version
2.4.2
```

Reply:
@jdblischak this can be when one of the dependencies does not have a R 3.4.1 version available. The rerenderer checks all dependencies.